docs/src/pages/components/tree-view/tree-view.md

```diff
@@ -62,3 +62,7 @@ const data = {
 (WAI-ARIA: https://www.w3.org/TR/wai-aria-practices/#TreeView)
 
 The component follows the WAI-ARIA authoring practices.
+
+To have an accessible tree view:
+
+- You must use `aria-labelledby` or `aria-label` to reference or provide a label on the TreeView.
```

Comment:
```suggestion
- You must use `aria-labelledby` or `aria-label` to reference or provide a label on the TreeView. Otherwise screen readers announce it as "tree", making it hard to understand the context of a specific treeitem for users who rely on screen readers.
```

which is similar to https://web.dev/button-name/. I think it's important to include the consequences so devs understand the "why" of the changes. Especially since the argument isn't as strong as labels for buttons. With buttons you don't have any context while for trees you get at least the name of a particular item. This may be enough to understand it.

Reply:
Suggestion to the suggestion:

"on the TreeView, otherwise screen readers will announce"

"context of a specific tree item ~~for users who rely on screen readers~~."

(I think the screen reader context is already established)